src/ui/sso/containers/consent.tsx

```diff
@@ -18,6 +18,9 @@ interface State {
 }
 
 export class ConsentContainer extends React.Component<Props, State> {
+  static navigationOptions = {
```

Comment:
What is the purpose of this?

Reply:
It's the title of the screen used by navigation. I guess we don't really need it